src/Connect.js

```diff
@@ -1,68 +1,508 @@
+   *  }
+   *  connect.createVerificationRequest(unsignedClaim).then(jwt => {
+   *    ...
+   *  })
+   *
+   *  @param    {Object}      reqObj                 object with request params
+   *  @param    {Object}      reqObj.unsignedClaim   an object that is an unsigned claim which you want the user to attest
+   *  @param    {String}      reqObj.sub             the DID which the unsigned claim is about
+   *  @param    {String}      [id='signClaimReq']    string to identify request, later used to get response
+   */
+  createVerificationRequest (reqObj, id = 'signClaimReq') {
+    this.credentials.createVerificationRequest(reqObj.unsignedClaim, reqObj.sub, this.genCallback(), this.did)
+      .then(jwt => this.request(jwt, id))
+  }
+
+  /**
+   *  Creates a [Selective Disclosure Request JWT](https://github.com/uport-project/specs/blob/develop/messages/sharereq.md)
+   *
+   *  @example
+   *  const req = { requested: ['name', 'country'],
+   *                callbackUrl: 'https://myserver.com',
+   *                notifications: true }
+   *  connect.requestDisclosure(req, 'REQUEST_ID')
+   *  connect.onResponse('REQUEST_ID').then(jwt => {
+   *      ...
+   *  })
+   *
+   *  @param    {Object}             [reqObj={}]           request params object
+   *  @param    {Array}              reqObj.requested      an array of attributes for which you are requesting credentials to be shared for
+   *  @param    {Array}              reqObj.verified       an array of attributes for which you are requesting verified credentials to be shared for
+   *  @param    {Boolean}            reqObj.notifications  boolean if you want to request the ability to send push notifications
+   *  @param    {String}             reqObj.callbackUrl    the url which you want to receive the response of this request
+   *  @param    {String}             reqObj.network_id     network id of Ethereum chain of identity eg. 0x4 for rinkeby
+   *  @param    {String}             reqObj.accountType    Ethereum account type: "general", "segregated", "keypair", or "none"
+   *  @param    {Number}             reqObj.expiresIn      Seconds until expiry
+   *  @param    {String}            [id='disclosureReq']   string to identify request, later used to get response
+   *  @return   {Promise<Object, Error>}                   a promise which resolves with a signed JSON Web Token or rejects with an error
+   */
+  requestDisclosure (reqObj, id = 'disclosureReq') {
+    reqObj = Object.assign({
+      accountType: this.accountType || 'none', 
+      callbackUrl: this.genCallback()
+    }, reqObj)
+    this.credentials.requestDisclosure(reqObj, reqObj.expiresIn)
+      .then(jwt => this.request(jwt, id))
+  }
+
+  /**
+   *  Create a credential about connnected user
+   *
+   *  @example
+   *  connect.attest({
+   *   sub: '5A8bRWU3F7j3REx3vkJ...', // uPort address of user, likely a MNID
+   *   exp: <future timestamp>,
+   *   claim: { name: 'John Smith' }
+   *  }, 'REQUEST_ID')
```

Comment:
Having the request id be `REQUEST_ID` is confusing. There is a default value, so lets remove it.